client/informant/src/lib.rs

```diff
@@ -20,18 +20,24 @@
 
 use ansi_term::Colour;
 use futures::prelude::*;
+use futures_timer::Delay;
 use log::{info, trace, warn};
 use parity_util_mem::MallocSizeOf;
 use sc_client_api::{BlockchainEvents, UsageProvider};
-use sc_network::NetworkStatus;
+use sc_network::NetworkService;
 use sp_blockchain::HeaderMetadata;
 use sp_runtime::traits::{Block as BlockT, Header};
 use sp_transaction_pool::TransactionPool;
-use sp_utils::{status_sinks, mpsc::tracing_unbounded};
+use sp_utils::mpsc::tracing_unbounded;
 use std::{fmt::Display, sync::Arc, time::Duration, collections::VecDeque};
 
 mod display;
 
+/// Creates a stream that returns a new value every `duration`.
+pub fn interval(duration: Duration) -> impl Stream<Item = ()> + Unpin {
```

Comment:
```suggestion
fn interval(duration: Duration) -> impl Stream<Item = ()> + Unpin {
```

If we make this public, it's only a matter of time before someone starts using `sc_informant::interval` :sweat_smile:

Reply:
By the way, I copied it from sc_network::utils::interval...